Added keystore config steps to Vulnerability detection and to Upgradi… (#7235)
wazuh/wazuh-documentation: source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst

      </synchronization>
    </wodle>

> Make sure you have saved the Wazuh indexer username and password into the Wazuh manager keystore using the Wazuh-keystore tool.

Reviewer: ```suggestion
You must save the Wazuh indexer username and password into the Wazuh manager keystore using the Wazuh-keystore tool.
```

Reviewer 2: There is no precedence or mandatory order, as long as both changes are applied before the Manager is restarted

.. code-block:: console
  

>   # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>

Reviewer: ```suggestion
   # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
```

>   # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_PASSWORD>

Reviewer: ```suggestion
   # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_PASSWORD>
```
